lib/fastlane/plugin/wpmreleasetoolkit/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fastlane
   module Wpmreleasetoolkit
-    VERSION = '8.1.0'.freeze
+    VERSION = '9.0.0'
```

Comment:
Rubocop complainted about this removing `.freeze` change.

Reply:
Thanks for the catch. Fixed in [1b2ceb8](https://github.com/wordpress-mobile/release-toolkit/pull/509/commits/1b2ceb8b2682195bc4ce91d7a23a763f81d0eb53) by using the `frozen_string_literal: true` magic comment on the whole file, so that we don't have to update our `rake new_release` script that changes this `VERSION = '…'` line on every release and risk encountering the same issue on next release.